daemon/pkg/registry/search_test.go

```diff
@@ -420,18 +420,6 @@ func TestSearch(t *testing.T) {
 func TestNewIndexInfo(t *testing.T) {
 	overrideLookupIP(t)
```

Comment:
Ah, yes; I was right on he stubbing out / mocking; https://github.com/moby/moby/blob/b26972f9f2d69d5064642b248d72c1696cbd8317/daemon/pkg/registry/registry_test.go#L12-L33

Reply:
These are the three that are failing; https://github.com/moby/moby/blob/b26972f9f2d69d5064642b248d72c1696cbd8317/daemon/pkg/registry/registry_test.go#L94-L96

The other two are here; common scenario looks to be matching against, e.g.`42.42.0.0/16`;

https://github.com/moby/moby/blob/b26972f9f2d69d5064642b248d72c1696cbd8317/daemon/pkg/registry/search_test.go#L633-L637